301-metadata-repositories.md

```diff
@@ -129,30 +207,12 @@ The following code listing is an example of the `targets` metadata for a bundle:
 }
 ```
 
-### Security analysis of MVP
-
-Table 1 presents a summary of possible attacks given the key compromise of one or more TUF roles and in-toto functionaries. We assume that attackers can:
-
-1. Compromise any combination of keys as per any row in Table 1.
-1. Serve new TUF and in-toto metadata as well as bundles. This can be done either with a man-in-the-middle (MitM) attack, or by compromising the metadata repository and CNAB registry.
-
-| TUF role, or in-toto functionary                             | Freeze attacks                    | Mix-and- match attacks            | Rollback attacks | Malicious bundles |
-|--------------------------------------------------------------|-----------------------------------|-----------------------------------|------------------|----------------------------|
-| root (TUF)                                                   | Yes                               | Yes                               | Yes              | Yes                        |
-| timestamp + snapshot (TUF)                                   | Yes (limited by root, or targets) | Yes (limited by root, or targets) | No               | No                         |
-| timestamp + snapshot + targets (TUF)                         | Yes (limited by TUF root)         | Yes                               | Yes              | Yes                        |
-| timestamp + snapshot + targets (TUF) + root layout (in-toto) | Yes (limited by TUF root)         | Yes                               | Yes              | Yes                        |
-| timestamp + snapshot + targets (TUF) + step1 (in-toto)       | Yes (limited by TUF root)         | Yes                               | Yes              | Yes                        |
-| timestamp + snapshot + targets (TUF) + step2 (in-toto)       | Yes (limited by TUF root)         | Yes                               | Yes              | Yes                        |
-
-**Table 1**: The security attacks that are possible given the key compromise of one or more TUF role or in-toto functionary. Columns marked "Yes" indicate that the specified attack is possible _given_ the compromise of _all_ of the specified keys, whereas columns marked "No" indicate that they are not.
+### Security analysis of extension
 
-As Table 1 suggests, bundle developers SHOULD use offline keys to sign:
+The biggest difference between this extension and the MVP itself is that there is now compromise-resilient distribution of the software supply chain for the bundle using the in-toto root layout.
```

Comment:
It might be good to summarize what this gives the user: "In addition to the assurances of the MVP, this provides additional assurance that..."